src/modules/api.js

```diff
@@ -0,0 +1,61 @@
+const API_URL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/nslYEPpt500fFqeTXs8e';
+
+async function submitScore(name, score) {
+  const response = await fetch(`${API_URL}/scores`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      user: name,
+      score,
+    }),
+  });
+  const result = await response.json();
+  return result;
+}
+
+async function getScores() {
+  const response = await fetch(`${API_URL}/scores`);
+  const result = await response.json();
+  return result;
+}
+
+const scoreForm = document.getElementById('score-form');
+scoreForm.addEventListener('submit', async (event) => {
+  event.preventDefault();
+  const nameInput = document.getElementById('name');
+  const scoreInput = document.getElementById('score');
+  const name = nameInput.value;
+  const score = scoreInput.value;
+  await submitScore(name, score);
+  nameInput.value = '';
+  scoreInput.value = '';
+});
+
+function createScoreRow(score) {
+  const row = document.createElement('tr');
+  const nameCell = document.createElement('td');
+  nameCell.textContent = score.user;
+  row.appendChild(nameCell);
+  const scoreCell = document.createElement('td');
+  scoreCell.textContent = score.score;
+  row.appendChild(scoreCell);
+  return row;
+}
```

Comment:
- [ ] According to the requirements we are supposed to use ES6 this includes usage of arrow functions instead of the `function keyword`, please refactor this function to an arrow function.